### PR TITLE
chore: remove background when stackPresentation is transparentModal

### PR DESCRIPTION
**Pull request by kaioduarte**
## Summary
Fixes https://github.com/react-navigation/react-navigation/issues/6877

## Before / After
<div style="display: flex;">

</div>

### CODE REVIEW

File: packages/native-stack/src/views/NativeStackView.tsx [hunk 1]
```diff
@@ -73,7 +73,12 @@ export default function NativeStackView({
             <View
               style={[
                 styles.container,
-                { backgroundColor: colors.background },
+                {
+                  backgroundColor:
+                    stackPresentation !== 'transparentModal'
```
Comment (bang88) [inline packages/native-stack/src/views/NativeStackView.tsx]: How about the old configuration `transparentCard:true`. Does this settings works too

Comment (kaioduarte) [reply packages/native-stack/src/views/NativeStackView.tsx]: This option is not related with the changed package (native-stack) and this option was [removed in v4](https://github.com/react-navigation/stack/blob/48b1651e99e3feb01c3caa5e0cab87b24b90ccf0/src/utils/validateDeprecatedConfig.tsx#L16).

**Comment by satya164** [comment]
Hey, thanks for the PR. We're moving the native-stack to screens repo: https://github.com/software-mansion/react-native-screens/pull/345

Would be great if you can send the PR there. Thanks and extremely sorry for the trouble.

**Comment by kaioduarte** [comment]
No problem, I'll manage to send there when your PR is merged. Thanks anyway!

**Pull request closed without merging**